global.json

```diff
@@ -1,6 +1,6 @@
 {
     "projects": [],
     "sdk": {
-        "version": "2.0.0"
+        "version": "2.1.4"
```

Comment:
I had to bump this up to get the ASP.NET Core projects to work at all in vs 15.6